frontend/views/home.ejs

```diff
@@ -3,21 +3,41 @@
 <head>
 	<link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Open+Sans">
 	<link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css' integrity='sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u' crossorigin='anonymous'>
-	<!-- <link rel='stylesheet' href='/style.css'> -->
+	<link rel='stylesheet' href='/style.css'>
 </head>
 
 <body>
 
-	<h1>Twitter Education bot</h1>
-	<p>Get tweets about class assignments coming up!</p>
+	<h1>Twitter Education Bot</h1>
+	<p>Sign up to get tweets about class assignments coming up!</p>
 
-	<h3>Register</h3>
-
-	<form action='register' method='POST'>
+	<form id="reg_form" action='register' method='POST'>
 		<input type='text' name='handle' placeholder='Twitter Handle'><br>
 		<input type='text' name='first_name' placeholder='First Name'><br>
 		<input type="text" name="last_name" placeholder='Last Name'><br>
-		<input type="text" name="enrolled" placeholder='Class'><br>
+		<!-- <input type="text" name="enrolled" placeholder='Class'><br> -->
+
+		<select name="enrolled[]" form="reg_form" size="5" multiple>
+			<option value="NA" selected>Select your classes</option>
+			<option value="161">161 - Intro to Comp Science I</option>
+			<option value="162">162 - Intro to Comp Science II</option>
+			<option value="165">165 - Accelerated intro to Comp Science</option>
```

Comment:
Same corrections as registered.ejs

Reply:
Thanks so much! I'll go ahead and fix those errors up!